src/components/button/_button.scss

```
@@ -515,6 +517,54 @@ $button-shadow-size: 3px;
             }
         }
     }
+
+    &--neutral & {
+        &__inner {
+            background: transparent;
+            border: 0;
+            border-radius: 0;
+            box-shadow: none;
+            color: var(--ons-color-black);
+            font-weight: $font-weight-regular;
+            padding: 2px;
+            .ons-icon {
+                fill: var(--ons-color-black);
+            }
+        }
+    }
+
+    &--neutral:hover &,
+    &--neutral:active &,
+    &--neutral.active & {
+        &__inner {
+            background: none;
+            color: var(--ons-color-text-link-hover);
+            .ons-icon {
+                fill: var(--ons-color-text-link-hover);
+            }
+        }
+    }
+
+    &--neutral:focus:hover & {
+        &__inner {
+            color: var(--ons-color-black);
+        }
+    }
+
+    &--neutral:focus &,
+    &--neutral:active:focus &,
+    &--neutral.active:focus & {
+        &__inner {
+            background-color: var(--ons-color-focus);
+            box-shadow:
+                0 -2px var(--ons-color-focus),
+                0 4px var(--ons-color-text-link-focus) !important;
+            color: var(--ons-color-text-link-focus);
+            .ons-icon {
+                fill: var(--ons-color-text-link-focus);
+            }
+        }
+    }
```

```suggestion
    &--neutral &,
    &--neutral:hover &,
    &--neutral:active &,
    &--neutral.active & {
        &__inner {
            background: transparent;
            border: 0;
            border-radius: 0;
            box-shadow: none;
            color: var(--ons-color-black);
            font-weight: $font-weight-regular;
            padding: 2px;
            .ons-icon {
                fill: var(--ons-color-black);
            }
        }
    }

    &--neutral:focus &,
    &--neutral:active:focus &,
    &--neutral.active:focus & {
        &__inner {
            background-color: var(--ons-color-focus);
            box-shadow:
                0 -2px var(--ons-color-focus),
                0 4px var(--ons-color-text-link-focus) !important;
            color: var(--ons-color-text-link-focus);
            .ons-icon {
                fill: var(--ons-color-text-link-focus);
            }
        }
    }
```

This can be simplified like this, avoid repetition and fix the hover and active blue link issues I've mentioned